Modules/_datetimemodule.c

```diff
@@ -3916,6 +3952,49 @@ time_new(PyTypeObject *type, PyObject *args, PyObject *kw)
     return self;
 }
 
+
+/* Return new time from time.strptime(). */
+static PyObject *
+time_strptime(PyObject *cls, PyObject *args)
+{
+    PyObject *time = NULL;
+    PyObject *datetime;
+
+    static PyObject *emptyDatetime = NULL;
+
+    /* To ensure that the given string does not contain a date,
+     * compare with the result of an empty date string.
+     */
+    if (emptyDatetime == NULL) {
+        PyObject *emptyStringPair = Py_BuildValue("ss", "", "");
+        if (emptyStringPair == NULL)
```

Comment:
If I'm interpreting [PEP 7](https://www.python.org/dev/peps/pep-0007/) correctly, these `if` statements need curly braces. The relevant section is [Code layout](https://www.python.org/dev/peps/pep-0007/#code-lay-out).